adapters/rubicon/rubicon.go

```diff
@@ -408,8 +409,8 @@ func (a *RubiconAdapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *ada
 				})
 				continue
 			}
-			bannerExt := rubiconBannerExt{RP: rubiconBannerExtRP{MIME: "text/html"}}
-			bannerCopy.Ext, err = json.Marshal(&bannerExt)
+			bannerExt := []byte(bannerExtContent) //this should be a constant outside of the function
+			bannerCopy.Ext = bannerExt
```

Comment:
Thank you for the change! Little correction is needed: please remove this comment, I added it in my code sample, but didn't mean to have it in final result ;)
Add cast to `[]byte` in Line 23 and then use it as is here: `bannerCopy.Ext = bannerExtContent`, no need to execute L412 every time.

Reply:
Nice catch)